### DISCUSSION TITLE
[Board]: Custom board with NRF

### DISCUSSION THREAD

**Discussion by Kralg** [meshtastic/firmware#2758] [General]
### SOC

NRF52

### Lora IC

nrf52840

### Product Link

https://eu.mouser.com/ProductDetail/Nordic-Semiconductor/nRF52840-QFAA-F-R7?qs=TCDPyi3sCW2TA2kPWr8xHQ%3D%3D

### Description

Hello,
We are designing the board and can you please write if the SX127x DIO0-DIO5 pins are crucial to be connected to MCU to run meshtastic on nrf or esp? We are using RFM95 module.

**Comment by code8buster** [comment]
Most 127x devices we support use DIO0-DIO2
Safe to say you should at least connect those three.
